### PR TITLE
[13.0][ADD] stock_inventory_justification

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

File: stock_inventory_justification/models/stock_inventory_justification.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class StockInventoryJustification(models.Model):
+    _name = "stock.inventory.justification"
+    _description = "Inventory justification"
+
+    _sql_constraints = [
+        (
+            "unique_name",
+            "EXCLUDE (name WITH =) WHERE (active = True)",
+            "This stock inventory justification already exsits.",
```
Comment (ntsirintanis) [inline stock_inventory_justification/models/stock_inventory_justification.py]: Please, let's try to fix the typo in this error message before merging, Overall, the module looks great, thanks for the effort!

Comment (ThomasBinsfeld) [reply stock_inventory_justification/models/stock_inventory_justification.py]: Done

**Review by ThomasBinsfeld — APPROVED** [review]
This one is ready for merge

**Review by hparfr — APPROVED** [review]

**Comment by rousseldenis** [comment]
@ntsirintanis Could you update your review ?

**Review by ntsirintanis — APPROVED** [review]

**Comment by ThomasBinsfeld** [comment]
Rebased on 13.0. Could we retry a merge if checks are ok?

**Review by HaraldPanten — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@rousseldenis I know this is being merged, but just wanted to ask: have you reviewed this alternative: [stock_change_qty_reason](https://github.com/OCA/stock-logistics-warehouse/tree/13.0/stock_change_qty_reason)?

**Pull request merged**